src/main.rs

```diff
@@ -176,10 +179,9 @@ fn shutdown_notifier(exit: Arc<AtomicBool>) -> io::Result<(Sender<()>, Receiver<
 }
 
 fn main() -> Result<(), ShredstreamProxyError> {
-    env_logger::builder()
-        .format_timestamp(Some(TimestampPrecision::Micros))
-        .init();
+    env_logger::builder().init();
```

Comment:
why get rid of micros resolution?

Reply:
we are mainly logging things every second or 10 seconds, didnt seem to be useful to have another 5 decimal places